assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -157,8 +202,14 @@ function FontPickerContainer({ value, onSelect, onClose }) {
 
   return (
     <PickerContainer ref={ref}>
+      <SearchInput
+        type="text"
+        value={searchKeyword}
+        onChange={handleSearchInputChanged}
+      />
```

Comment:
This definitely needs a label, but maybe also a placeholder? Also, `type="search"` might be an idea, as it changes the iOS keyboard to have a "search" button rather than an "enter" button.

Reply:
Agreed. I didn't see one or a clear button in the UX, but I'd assume we'd want to add that as we iterate this.

Reply:
<img width="218" alt="Screen Shot 2020-06-08 at 5 50 31 PM" src="https://user-images.githubusercontent.com/1738349/84087813-948f9f00-a9b0-11ea-8b32-e56f273edf18.png">

I did not realize Chrome added a clear button to `type: search`. This makes that easier.